runbotics/runbotics-sdk/src/handler/DesktopRunRequest.ts

```diff
@@ -8,6 +8,16 @@ interface CommonDesktopRunRequest {
     trigger: { name: string };
     parentProcessInstanceId?: string;
     triggerData?: unknown;
+    credentials: {
+        id: string;
+        name: string;
+        template: string;
+        attributes: {
+            id: string;
+            name: string;
+            value: string;
+        }[];
+    }[];
```

Comment:
If i remember correctly @Aeri4a created credential interface in rb-common - credential should be defined in one place, so can be reused, and any changes would appear in the whole project. If you're merging before Ania, leave @TODO comment